frontend/src/components/ViewWrappers/ViewWrappers.scss

```diff
@@ -0,0 +1,16 @@
+@import "stylebase/variables";
+
+
+.View--padded,
+.ViewSplitter--padded {
+  padding: $padding-wrapper;
+}
+
+.View {
+  display: flex;
+  height: 100%;
+}
+
+.ViewSplitter {
+  flex: 1 1 100%;
```

Comment:
A border between the split view would look a bit nicer:

```
.ViewSplitter {
  flex: 1 1 100%;

  & + & {
    border-left: 1px solid $color-primary;
  }
}
```

The `& + &` shorthand is basically just saying "all elements except the first".